### PR TITLE
Update ppp for developer Ou Chuhao

### PR DESCRIPTION
**Pull request by DerrickSaltFish**
Update ppp for Ou Chuhao;
Fixed bugs in index.md.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ title: Intern Buddy
 
 ![Ui](images/Ui.png)
 
-**InternBuddy is a desktop application for for Computing undergraduates to manage and track their internship applications.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**InternBuddy is a desktop application for Computing undergraduates to manage and track their internship applications.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
 
 * If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
 * If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
```
Comment (seadragon2000341) [inline docs/index.md:14]: Same here

File: docs/index.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ title: Intern Buddy
 
 ![Ui](images/Ui.png)
 
-**InternBuddy is a desktop application for for Computing undergraduates to manage and track their internship applications.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**InternBuddy is a desktop application for Computing undergraduates to manage and track their internship applications.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
 
 * If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
```
[the remark this answers is not in the document]
Comment (DerrickSaltFish) [reply docs/index.md:13]: Noted

[the remark this answers is not in the document]
Comment (DerrickSaltFish) [reply docs/index.md:13]: > 

Noted

File: docs/index.md (hunk 1 above)
Comment (DerrickSaltFish) [reply docs/index.md]: > Same here

Modified in the newest commit

File: docs/team/derricksaltfish.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+InternBuddy provides a 1-stop platform for Computing undergraduates to manage and track their internship applications. It is optimized for typing which Computing undergraduates are comfortable and proficient in, allowing them to fully and efficiently exploit the application’s organisational capabilities.
+The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+
+Given below are my contributions to the project.
+
+* **New Feature**: [To be added]
+* **New Feature**: [To be added]
+
+* **Code contributed**: [RepoSense link]()
```
[the remark this answers is not in the document]
Comment (DerrickSaltFish) [reply docs/team/derricksaltfish.md:16]: > Reposense link not correct

Do you know where to find my reposense page?

[the remark this answers is not in the document]
Comment (DerrickSaltFish) [reply docs/team/derricksaltfish.md:16]: > Reposense link not correct

Modified in the latest commit

**Comment by eugenetangkj** [comment]
Looks good to merge

**Pull request merged**